source/MaterialXTest/GenReference.cpp

```diff
@@ -142,7 +232,7 @@ TEST_CASE("GenShader: OSL Reference", "[genshader]")
         stdlibDoc->removeChild(node->getName());
     }
 
-    mx::writeToXmlFile(implDoc, outputPath / "stdlib_osl_impl.mtlx");
+    mx::writeToXmlFile(implDoc, outputPath / "stdlib_osl_impl.refmtlx");
```

Comment:
@niklasharrysson : To avoid accidental usage of reference library, just rename this for now to avoid scanning this in for tests which come after this one (like render tests).